libcnb/src/runtime.rs

```diff
@@ -200,13 +200,13 @@ fn parse_build_args_or_exit() -> BuildArgs {
     }
 }
 
-fn read_buildpack_dir<E: Display + Debug>() -> Result<PathBuf, E> {
+fn read_buildpack_dir<E: Debug>() -> crate::Result<PathBuf, E> {
```

Comment:
I presume the `s/Result/crate::Result/` is to prevent confusion with the stdlib `Result`. Should the `use crate::Result;` in this file be removed to force all usages to specify `crate::Result`?

Reply:
Yeah, that's the intention. I like your idea and think we should apply this to the whole codebase! I filed #185 for it and will merge this as-is for now to keep this PR focussed on the actual change.